src/main/java/seedu/myledger/MyLedger.java

```diff
@@ -44,10 +44,10 @@ public static void readUserInputs() {
 
     public static boolean hasProcessedAllInputs(String line, ExpenditureList expenditures) {
         // Parses the input
+        ExpenditureList.queryLumpSumDates();
```

Comment:
It was advised to run this method before command execution, however after some consideration, we will run this method twice. Once after the expenditures have been read off the saved file and the second continuously in the main loop of the program. For the latter we will have the method ran before saving the list in the main program so that the repeated date can be saved immediately after user input